api/src/repositories/security-repository.ts

```diff
@@ -0,0 +1,124 @@
+import SQL from 'sql-template-strings';
+import { z } from 'zod';
+import { ApiExecuteSQLError } from '../errors/api-error';
+import { getLogger } from '../utils/logger';
+import { BaseRepository } from './base-repository';
+
+const defaultLog = getLogger('repositories/security-repository');
+
+export const PersecutionAndHarmSecurity = z.object({
```

Comment:
I followed example in code, 'artifact-repository.ts' 
![image](https://github.com/bcgov/biohubbc-platform/assets/35311998/ee2ac98a-92d0-41d0-92d3-51666c905589)
Updating these variable names causes slight issues, as the prefix `I` causes a issues by type.
![image](https://github.com/bcgov/biohubbc-platform/assets/35311998/5bf7e533-fb94-40cc-bdf9-697059eaeab9)
Where Zod is maintaining our json object restriction, naming I on the prefix no longer allows zod to be an object.

Reply:
The `I` prefix shouldn't make or break anything, its just a stylistic choice.

We originally used it for interfaces, just because it was a pattern others were using. But for types, it starts to break down a bit, since its not technically an interface (even though the difference is usually negligible).

But ya, I don't have a strong preference personally and find myself using both at times. As long as the type is named appropriately that is what matters more. If anyone has an opinion one way or the other though, then we can do that.